aws/resource_aws_iam_role.go

```diff
@@ -105,6 +105,24 @@ func resourceAwsIamRole() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+
+			"max_session_duration": {
+				Type:     schema.TypeInt,
+				Optional: true,
+				Default:  3600,
+				ValidateFunc: func(v interface{}, k string) (ws []string, errors []error) {
+					value := v.(int64)
```

Comment:
how about replace custom function with existing validation.IntBetween() instead?

Reply:
As noted above, this should be updated to `ValidateFunc: validation.IntBetween(3600, 43200)`